lib/Doctrine/Inflector/InflectInterface.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Inflector;
+
+interface InflectInterface
```

Comment:
Should not be suffixed with `Interface` and class name imho shouldn't be a verb (what's wrong with `Inflector`?

Reply:
Right now I have left the old Inflector class with the static methods and have maintained BC with the old interface proxying to the new one. Once I have the tests moved over to the new API I will remove the existing Inflector class and rename things.